ReroSlashCommands.cs

```diff
@@ -0,0 +1,205 @@
+using System.Text.RegularExpressions;
+using DSharpPlus;
+using DSharpPlus.Entities;
+using DSharpPlus.SlashCommands;
+using Microsoft.Data.Sqlite;
+
+namespace ReactionRolesBotCS;
+
+[SlashCommandGroup("reactionroles", "Commands related to reaction roles")]
+public class ReroSlashCommands : ApplicationCommandModule
+{
+
+	/// <summary>
+	///		Add Rero Command.
+	/// </summary>
+	/// <param name="ctx"></param>
+	/// <param name="emoji"></param>
+	/// <param name="role"></param>
+	/// <returns></returns>
```

Comment:
What is with all these XML looking comments?

Reply:
Standard C# method documentation

Reply:
If it's standard then they can stay.